ui/src/infra-model/translations.fi.json

```diff
@@ -85,7 +85,8 @@
             "measurementMethod": "Laatu",
             "coordinateSystemSrid": "Koordinaattijärjestelmä",
             "verticalCoordinateSystem": "Korkeusjärjestelmä",
-            "error-message": "Tiedostossa on virheitä"
+            "error-message": "Tiedostossa on virheitä",
+            "createdDate": "Suunnitelman luontiaika puuttuu"
```

Comment:
Täällä voisi varmaan erottaa nämä camelCasella olevat header-kentät oman käännösavaimensa taakse, sillä tämä näyttää vähän jännältä näin. Ei tosin mikään iso juttu vaikka nämä ovat näinkin, näyttää vaan vähän jännältä

Reply:
Joo hyvä idea. Nuo tulee varmaan jostain kentän json-nimen käyttämisestä avaimena?

Reply:
jep